Firestore/core/src/firebase/firestore/remote/existence_filter.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2019 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_REMOTE_EXISTENCE_FILTER_H_
+#define FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_REMOTE_EXISTENCE_FILTER_H_
+
+namespace firebase {
+namespace firestore {
+namespace remote {
+
+/** Simplest form of existence filter */
```

Comment:
The comment is from Android, not sure if it adds that much value, though.

Reply:
The ts port additionally references b/33078163. According to that, there was originally a desire to support bloom filters too... but that was marked obsolete.

Reply:
Removed.